include/seastar/util/log.hh

```diff
@@ -322,6 +322,58 @@ public:
     }
     /// \endcond
 
+#if FMT_VERSION >= 80000
+    /// Log with error tag
+    ///
+    /// \param s - message to be printed
+    ///
+    /// \sa logger::error(fmt_string<Args...> fmt, Args&&... args) noexcept
+    void error(std::string_view s) noexcept {
+        log(log_level::error, "{}", s);
+    }
+    /// Log with warn tag
+    ///
+    /// \param s - message to be printed
+    ///
+    /// \sa logger::warn(fmt_string<Args...> fmt, Args&&... args) noexcept
+    void warn(std::string_view s) noexcept {
+        log(log_level::warn, "{}", s);
+    }
+    /// Log with info tag
+    ///
+    /// \param s - message to be printed
+    ///
+    /// \sa logger::info(fmt_string<Args...> fmt, Args&&... args) noexcept
+    void info(std::string_view s) noexcept {
+        log(log_level::info, "{}", s);
+    }
+    /// Log with info tag on shard zero only:
+    ///
+    /// \param s - message to be printed
+    ///
+    /// \sa logger::info0(fmt_string<Args...> fmt, Args&&... args) noexcept
+    void info0(std::string_view s) noexcept {
+        if (is_shard_zero()) {
+            log(log_level::info, "{}", s);
+        }
+    }
+    /// Log with debug tag
+    ///
+    /// \param s - message to be printed
+    ///
+    /// \sa logger::debug(fmt_string<Args...> fmt, Args&&... args) noexcept
+    void debug(std::string_view s) noexcept {
+        log(log_level::debug, "{}", s);
+    }
+    /// Log with trace tag
+    ///
+    /// \param s - message to be printed
+    ///
+    /// \sa logger::trace(fmt_string<Args...> fmt, Args&&... args) noexcept
+    void trace(std::string_view s) noexcept {
+        log(log_level::trace, "{}", s);
+    }
+#endif
```

Comment:
Should be folded into the previous patch, or it creates a temporary regression.

What happens if I write: `logger.warn("dsfasf {} sdfas")? Does it use the string_view overload and compile, or does it fail at compile time as it should?

Reply:
right. folded.

Reply:
Avi's question about what happens to a string constant is a good one. 
Would have been good to test this in the unit test to avoid regressions - but it might be impossible (or exceedingly difficult) to check a compile-time error :-(

Reply:
> What happens if I write: `logger.warn("dsfasf {} sdfas")? Does it use the string_view overload and compile, or does it fail at compile time as it should?

not sure if this is a rhetorical question. if it is not, i think i am following @nyh 's proposal. so if the code is compiled with fmt v8 and up, the `string_view` overload would be used. but it just fails to build if compiled with fmt < 8.0, as tested by the conditionally disabled test.

> Would have been good to test this in the unit test to avoid regressions - but it might be impossible (or exceedingly difficult) to check a compile-time error :-(

good point, i can add a test with the help of cmake if no objections. something like (not tested):
```cmake
add_executable(test_unit_fmt-string-check
  EXCLUDE_FROM_ALL
  log_with_bad_fmt.cc)
target_link_libraries(test_unit_fmt-string-check
  seastar_testing
  seastar_private)
add_test(
  NAME Seastar.unit.fmt-string-check
  COMMAND ${CMAKE_COMMAND} --build . --target test_unit_fmt-string-check --config $<CONFIGURATION>
  WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
set_tests_properties(Seastar.unit.fmt-string-check
  PROPERTIES
    WILL_FAIL TRUE)
```
 i understand that we are not quite in favor of CMake. but without the help of it, it'd be difficult to implement such a test.